lib/rubyblok/helpers/storyblok_helper.rb

```diff
@@ -35,13 +36,28 @@ def rubyblok_blocks_tag(bloks)
     render_inline_partial template:, locals: { bloks: }
   end
 
+  def rubyblok_link_tag(text, content)
+    template =
+      %(<%= link_to text, content.url %>)
+
+    render_inline_partial template:, locals: { content:, text: }
+  end
+
+  def rubyblok_image_tag(content, **options)
```

Comment:
the input parameter should be the asset option hash provided by storyblok, it is something like
```
"asset"=>
        {"id"=>14139998,
         "alt"=>"",
         "name"=>"",
         "focus"=>"",
         "title"=>"",
         "source"=>"",
         "filename"=>"https://a.storyblok.com/f/236225/768x359/40dbc6e79a/rubyblok-product-chart-4-v2.svg",
         "copyright"=>"",
         "fieldtype"=>"asset",
         "meta_data"=>{},
         "is_private"=>false,
         "is_external_url"=>false},
```